packages/app/src/resource/BotEditor.tsx

```diff
@@ -49,6 +50,7 @@ export function BotEditor(): JSX.Element | null {
       .readResource('Bot', id)
       .then(async (newBot: Bot) => {
         setBot(newBot);
+        setModule(newBot.runtimeVersion === 'vmcontext' ? 'commonjs' : 'esnext');
```

Comment:
Do we know if fission will also work with `esnext`?

Or should we invert this check?
```suggestion
        setModule(newBot.runtimeVersion === 'awslambda' ? 'esnext' : 'commonjs');
```

Reply:
Good call - we should test.